integration-tests/return-response/src/test/java/io/quarkiverse/openapi/generator/it/ReturnResponseTest.java

```diff
@@ -4,8 +4,7 @@
 
 import jakarta.ws.rs.core.Response;
 
-import org.acme.openapi.api.ModelApi;
-import org.acme.openapi.api.ResponseApi;
+import org.acme.openapi.api.*;
```

Comment:
Please don't use star imports.
You can run `mvn verify -DskipTests` before committing to format and organize imports.